uptane-standard.md

```diff
@@ -839,26 +849,26 @@ In order to perform full verification, an ECU SHALL perform the following steps:
 1. Download and check the Timestamp metadata file from the Image repository, following the procedure in {{check_timestamp}}.
 1. Download and check the Snapshot metadata file from the Image repository, following the procedure in {{check_snapshot}}.
 1. Download and check the top-level Targets metadata file from the Image repository, following the procedure in {{check_targets}}.
-1. Verify that Targets metadata from the Director and Image repositories match. A primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the target metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
+1. Verify that Targets metadata from the Director and Image repositories match. A Primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A Secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the Targets metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
```

Comment:
In the parenthetical here, it actually is "target metadata" we're talking about, not "Targets metadata". That is, it's the metadata about the specific target within the Targets metadata. That's definitely not very clear, though. We could change it to something like `(That is, the metadata for the particular target within the Targets metadata from the Director that contains the ECU identifier of the current ECU.)` Might be a bit wordy, though.

Reply:
I hesitated on changing this because I thought it might be referring a target in the generic sense. Let's keep it simple. I'll put it back to the original.

Reply:
FIXED.